### DISCUSSION TITLE
Seeking Insights: Convincing Management to Shift to the Vendure Platform

### DISCUSSION THREAD

**Discussion by zulfi007** [vendurehq/vendure#2357] [General]
Hello everyone,

I'm looking for insights to help convince our management about shifting to the Vendure platform for B2B + B2C Commerce. Besides our team's JavaScript and Python expertise, and Vendure's flexibility, what other key points can I emphasize? Specifically, I'm interested in addressing:

- Community Support
- Constant Development
- Features from a non-technical angle
- Stability, Reliability, Security
- Ecosystem and Plugins
- Case Studies / Success Stories
- Total Cost of Ownership analysis
- Non-tech Staff Learning Curve

I appreciate your perspectives on these vital non-technical aspects, which are essential for a successful platform shift.

**Comment by michaelbromley** [comment]
Hi!

What platform would you be moving away from? I'll share my thoughts on each of those areas of interest:

> Community Support

We have an active [Discord community](https://vendure.io/community) where you & your team can get help if you run into any problems. I also hang out there and can directly help when I have time. Furthermore we have an active community of developers & agencies who can assist in development or consulting if needed.

> Constant Development

Well, take a look at the [commit activity](https://github.com/vendure-ecommerce/vendure/graphs/commit-activity) as well as the [regular release cadence](https://github.com/vendure-ecommerce/vendure/blob/master/CHANGELOG_v1.md) - there has been constant, steady development over the past 5 years and that is set to continue.

> Features from a non-technical angle

This is something we can improve our marketing on, but e.g. I'd say some stand-out non-technical features are the built-in multi-language support, the flexible dynamic collections, the powerful promotions system, built-in asset management. 

> Stability, Reliability, Security

This is really a function of your own team's operations ability. But from our side, I can say that Vendure is very well tested in many production applications, some very large with high traffic loads. We're over 2 years since our v1.0 release so stability is at a pretty good point now. We have several large enterprises / startups building on Vendure, some of which use it as the entire basis of their main product. So it does get a good deal of battle-testing.

> Ecosystem and Plugins

We have an active community providing plugins such as [pinelab plugins](https://pinelab-plugins.com/) and also on our [marketplace page](https://www.vendure.io/marketplace) and we plan to expand the number of integrations for things like taxes, payments, shipping, CMS, ERP, PIM etc.

> Case Studies / Success Stories

- Case studies: https://www.vendure.io/customers
- Our [showcase channel on Discord](https://vendure.io/community) has more live examples
- There are a number of non-public projects that I can share if you need them if you DM me on Discord.

> Total Cost of Ownership analysis

I don't have any concrete figures on this. Since the platform itself is free an open source, the vast majority of your costs will be in the form of development time. Since you mention you already have a team of devs, then the next cost will be hosting and that very much depends on the kind of loads you need to handle.

> Non-tech Staff Learning Curve

We've got some guides here: https://docs.vendure.io/user-guide/ but we plan on expanding these to cover much more and update with the latest features in the Admin UI.
